### PR TITLE
SLO Dashboard Log Search functionality

### PR DESCRIPTION
**Pull request by kimsauce**
## Purpose of this pull request

This pull request (PR) ...

Issue number: 

## Select the type of change:

- [ ] Minor Changes - Typos, formatting, slight revisions
- [ ] Update Content - Revisions and updating sections
- [x] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, and new packages for the site, Gatsby, React, etc

### CODE REVIEW

File: docs/observability/reliability-management-slo/dashboards.md [hunk 1]
```diff
@@ -62,12 +62,23 @@ Each dashboard contains the following information:<br/><img src={useBaseUrl('img
 </table>
 
 
-## Setting Granularity for SLO Time Ranges
+## Setting SLO Data Granularity
+
+### Time Ranges
 
 To modify the time range, select and drag across dates to zoom in further. This can be useful if you want to zoom in for granular details, especially for charts with larger compliance periods:<br/><img src={useBaseUrl('img/observability/2022-08-02_14-29-19.gif')} alt="Reliability Management SLO SLI" />
```
Comment (JainM6) [inline docs/observability/reliability-management-slo/dashboards.md:69]:
```suggestion
To modify the time range, select and drag across dates to zoom in further. This can be useful if you want to zoom in for granular details, especially for charts with larger compliance periods.<br/><img src={useBaseUrl('img/observability/2022-08-02_14-29-19.gif')} alt="Reliability Management SLO SLI" />
```

Comment (JainM6) [reply docs/observability/reliability-management-slo/dashboards.md:69]: Removed  `:`

File: docs/observability/reliability-management-slo/dashboards.md [hunk 2]
```diff
@@ -62,12 +62,23 @@ Each dashboard contains the following information:<br/><img src={useBaseUrl('img
...
 
+### Compliance Period
+
 You can also filter by compliance period to view your past activity and plan ahead:<br/><img src={useBaseUrl('img/observability/compliance-period-filter.png')} alt="Reliability Management SLO SLI" />
```
Comment (JainM6) [inline docs/observability/reliability-management-slo/dashboards.md:73]:
```suggestion
You can also filter by compliance period to view your past activity and plan ahead.<br/><img src={useBaseUrl('img/observability/compliance-period-filter.png')} alt="Reliability Management SLO SLI" />
```

Comment (JainM6) [reply docs/observability/reliability-management-slo/dashboards.md:73]: @kimsauce: Is there any specific reason for using ":" here? I see it in the above sentence as well.

File: docs/observability/reliability-management-slo/dashboards.md [hunk 3]
```diff
@@ -62,12 +62,23 @@ Each dashboard contains the following information:<br/><img src={useBaseUrl('img
...
 
+### Accessing Log Search
+
+You can launch a Log Search session directly from an SLO dashboard panel, giving you the ability to drill down into further granular details.
+
```
Comment (JainM6) [inline docs/observability/reliability-management-slo/dashboards.md:78]:
```suggestion
```

Comment (JainM6) [inline docs/observability/reliability-management-slo/dashboards.md:78]:
```suggestion
```

File: docs/observability/reliability-management-slo/dashboards.md [hunk 4]
```diff
@@ -62,12 +62,23 @@ Each dashboard contains the following information:<br/><img src={useBaseUrl('img
...
+1. Go to **Manage Data** > **Monitoring** > **SLO**.
+1. Click on any SLO line item.
+1. Hover over the panel > click the three-dot kebab icon > **Open in Log Search**.<br/><img src={useBaseUrl('img/observability/open-in-logsearch.png')} alt="open-in-logsearch" width="150"/>
```
Comment (JainM6) [inline docs/observability/reliability-management-slo/dashboards.md:81]: I'd rephrase this as:
```suggestion
1. Hover over the panel, click the three-dot kebab icon, and select the **Open in Log Search** option.<br/><img src={useBaseUrl('img/observability/open-in-logsearch.png')} alt="open-in-logsearch" width="150"/>
```

**Review by JainM6 — DISMISSED** [review]
Minor Formatting changes

File: blog-service/2023-02-07-slo-log-search.md [hunk 5]
```diff
@@ -0,0 +1,18 @@
...
+  - dashboards
+hide_table_of_contents: true
+authors:
+  - url: https://help.sumologic.com/release-notes-service/rss.xml
+    image_url: /img/release-notes/rss-orange.png
+---
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+You can now launch a Log Search session directly from an SLO dashboard panel, giving you the ability to drill down further into your SLO data. [Learn more](https://help.sumologic.com/docs/observability/reliability-management-slo/dashboards).
```
Comment (apoorv-garg) [inline blog-service/2023-02-07-slo-log-search.md:16]: We can add - `You can further add the log search results into any other dashboard, allowing you to correlate your SLO performance with other data`.

File: docs/observability/reliability-management-slo/dashboards.md [hunk 6]
```diff
@@ -62,12 +62,23 @@ Each dashboard contains the following information:<br/><img src={useBaseUrl('img
...
+
+### Time Ranges
 
 To modify the time range, select and drag across dates to zoom in further. This can be useful if you want to zoom in for granular details, especially for charts with larger compliance periods:<br/><img src={useBaseUrl('img/observability/2022-08-02_14-29-19.gif')} alt="Reliability Management SLO SLI" />
 
+### Compliance Period
+
 You can also filter by compliance period to view your past activity and plan ahead:<br/><img src={useBaseUrl('img/observability/compliance-period-filter.png')} alt="Reliability Management SLO SLI" />
 
+### Accessing Log Search
```
Comment (apoorv-garg) [inline docs/observability/reliability-management-slo/dashboards.md:75]: how about `Open In Log Search`

File: docs/observability/reliability-management-slo/dashboards.md (hunk 2 above)
Comment (kimsauce) [reply docs/observability/reliability-management-slo/dashboards.md:73]: Colon precedes an image, like this:
Screenshot 2023-02-07 at 12 20 22 PM

File: docs/observability/reliability-management-slo/dashboards.md [hunk 7]
```diff
@@ -62,12 +62,23 @@ Each dashboard contains the following information:<br/><img src={useBaseUrl('img
...
-To modify the time range, select and drag across dates to zoom in further. This can be useful if you want to zoom in for granular details, especially for charts with larger compliance periods:<br/><img src={useBaseUrl('img/observability/2022-08-02_14-29-19.gif')} alt="Reliability Management SLO SLI" />
+### Time Ranges
+
+To modify the time range, select and drag across dates to zoom in further. This can be useful if you want to zoom in for granular details, especially for charts with larger compliance periods.<br/><img src={useBaseUrl('img/observability/2022-08-02_14-29-19.gif')} alt="Reliability Management SLO SLI" />
+
+### Compliance Period
 
 You can also filter by compliance period to view your past activity and plan ahead:<br/><img src={useBaseUrl('img/observability/compliance-period-filter.png')} alt="Reliability Management SLO SLI" />
 
+### Accessing Log Search
```
Comment (kimsauce) [inline docs/observability/reliability-management-slo/dashboards.md:75]:
```suggestion
### Open in Log Search
```

File: docs/observability/reliability-management-slo/dashboards.md (hunk 4 above)
Comment (kimsauce) [reply docs/observability/reliability-management-slo/dashboards.md:80]: I'll add to next meeting agenda. Typically when introducing cascading UI elements, most tech docs use `>` (visually, it's easier to scan).

File: blog-service/2023-02-07-slo-log-search.md (hunk 5 above)
Comment (kimsauce) [inline blog-service/2023-02-07-slo-log-search.md:16]:
```suggestion
You can now launch a Log Search session directly from an SLO dashboard panel, giving you the ability to drill down further into your SLO data. You can also add Log Search results to any other dashboard, allowing you to correlate SLO performance data with other categories of data. [Learn more](https://help.sumologic.com/docs/observability/reliability-management-slo/dashboards).
```

File: blog-service/2023-02-07-slo-log-search.md [hunk 8]
```diff
@@ -0,0 +1,18 @@
...
+  - dashboards
+hide_table_of_contents: true
+authors:
+  - url: https://help.sumologic.com/release-notes-service/rss.xml
+    image_url: /img/release-notes/rss-orange.png
+---
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+You can now launch a Log Search session directly from an SLO dashboard panel, giving you the ability to drill down further into your SLO data. You can also add Log Search results to any other Sumo Logic dashboard, allowing you to correlate SLO performance data with other categories of data. [Learn more](https://help.sumologic.com/docs/observability/reliability-management-slo/dashboards).
```
Comment (kimsauce) [inline blog-service/2023-02-07-slo-log-search.md:16]:
```suggestion
You can now launch a Log Search session directly from an SLO dashboard panel, allowing you to drill down further into your SLO data. With this feature, you can also add Log Search results to any other Sumo Logic dashboard, allowing you to correlate SLO performance data with other categories of data. [Learn more](https://help.sumologic.com/docs/observability/reliability-management-slo/dashboards).
```

File: docs/observability/reliability-management-slo/dashboards.md [hunk 9]
```diff
@@ -62,12 +62,22 @@ Each dashboard contains the following information:<br/><img src={useBaseUrl('img
...
+### Compliance Period
+
+You can also filter by compliance period to view your past activity and plan ahead.<br/><img src={useBaseUrl('img/observability/compliance-period-filter.png')} alt="Reliability Management SLO SLI" />
+
+### Open in Log Search
+
+You can launch a Log Search session directly from an SLO dashboard panel, giving you the ability to drill down into further granular details.
+1. Go to **Manage Data** > **Monitoring** > **SLO**.
+1. Click on any SLO line item.
+1. Hover over the panel, click the three-dot kebab icon, and select the **Open in Log Search** option.<br/><img src={useBaseUrl('img/observability/open-in-logsearch.png')} alt="open-in-logsearch" width="150"/>
```
Comment (kimsauce) [inline docs/observability/reliability-management-slo/dashboards.md:80]:
```suggestion
1. Double-click on any SLO line item.
1. Hover over the panel > Click the kebab icon > **Open in Log Search**.<br/><img src={useBaseUrl('img/observability/open-in-logsearch.png')} alt="open-in-logsearch" width="150"/>
```

**Review by stacykor — APPROVED** [review]
LGTM

File: docs/observability/reliability-management-slo/dashboards.md (hunk 2 above)
Comment (JainM6) [reply docs/observability/reliability-management-slo/dashboards.md]: That's a complete sentence and we can't put colon here. It's rule in grammar and the reason is colon and em dashes are always used to signal further clarification or used with lists.

Comment (kimsauce) [reply docs/observability/reliability-management-slo/dashboards.md]: Per the example in my previous comment, this sentence (and the ones like it throughout the docs) are introducing images and code snippets. This formatting is used everywhere in tech docs (Google, Microsoft, GitHub, etc.). Can you please add to Docs Meeting agenda for discussion?
